elements.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: default
+title: Atomic Elements
+---
+
+# Atomic Elements
+
+This is a cheat sheet for the Atomic Elements used in the [Element constructor](chemistry-lab#element-constructor)
+
+Please note, the Protons and Electrons are the same number and are based on the Atomic Number.
+
+Atomic Number = Protons = Electrons
```

Comment:
Maybe just add an example to be super clear?
`e.g. Lithium has 3 protons, 4 neutrons and 3 electrons`